src/strategies/proxy-erc1155-balance-of/index.ts

```diff
@@ -0,0 +1,102 @@
+import fetch from 'cross-fetch';
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'rawrjustin';
+export const version = '0.1';
+
+const abi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'owner',
+        type: 'address'
+      },
+      {
+        internalType: 'uint256',
+        name: 'id',
+        type: 'uint256'
+      }
+    ],
+    name: 'balanceOf',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+const calculateVotingPower = (inputAddresses, addressScores, walletMap) => {
+  let userVotingPower = {};
+  inputAddresses.forEach(input => {
+    let count = 0.0
+    walletMap[input.toLowerCase()].forEach(address => {
+      count += addressScores[address]
+    });
+    userVotingPower[input] = count
+  });
+  return userVotingPower
+};
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  // Get the wallet mapping from proxy wallets to actual wallets
+  const url = 'https://api.proxychat.xyz/external/v0/getProxyWalletMappings';
```

Comment:
What does this service do?

Reply:
the service is a community management tool that allows communities to have social functionality on top of snapshot and other governance features like idea boards/ forums. https://app.proxychat.xyz/post/PN7bFYYikNqD58s74x5F/

we want to allow users to vote with a proxy wallets so that you don't need to sign using your cold wallet everytime you want to vote with your NFT. 

the mapping returned here is a mapping of a temporary proxy wallets -> actual wallets that hold tokens/nfts as detailed in the example. Let me know if that makes sense!

Reply:
the service is a community management tool that allows communities to have social functionality on top of snapshot and other governance features like idea boards/ forums. https://app.proxychat.xyz/post/PN7bFYYikNqD58s74x5F/

we want to allow users to vote with a proxy wallets so that you don't need to sign using your cold wallet everytime you want to vote with your NFT. 

the mapping returned here is a mapping of a temporary proxy wallets -> actual wallets that hold tokens/nfts as detailed in the example. Let me know if that makes sense!

Reply:
Awesome 👍 actual problem is that these strategies sound like common strategies. maybe can rename as `proxychat-*` will help reduce confusion with keyword `proxy`

Reply:
Also would be great if you can create separate PRs for each strategy :) it will do great help with testing/review/integration/support etc . Thanks